engine/gui/src/gui_script.cpp

```diff
@@ -4189,6 +4234,8 @@ namespace dmGui
         {"pick_node",       LuaPickNode},
         {"is_enabled",      LuaIsEnabled},
```

Comment:
This is the exception. Should probably be deprecated, and add a "get_enabled()" for consistency.
It was added in 2012, but after that, all other bool-getters were named "get", e.g. the ones from Ragnar in 2014.